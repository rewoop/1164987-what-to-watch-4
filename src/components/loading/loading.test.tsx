import React from "react";
import renderer from "react-test-renderer";
import Loading from "./loading.jsx";

it(`Should Loading render correctly`, () => {
  const tree = renderer
    .create(<Loading />).toJSON();

  expect(tree).toMatchSnapshot();
});
