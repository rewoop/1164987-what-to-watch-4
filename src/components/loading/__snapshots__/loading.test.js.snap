// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Loading render correctly 1`] = `
<div
  className="sweet-loading"
>
  <div
    className="css-ymva0y-override"
  >
    <div
      className="css-1ittl2f"
    />
    <div
      className="css-8wwzaj"
    />
    <div
      className="css-p4djol"
    />
    <div
      className="css-12wxdfg"
    />
    <div
      className="css-p0n8q2"
    />
    <div
      className="css-13n574i"
    />
  </div>
</div>
`;
